### PR TITLE
Correct styles for selected items

### PR DESCRIPTION
**Pull request by micgro42**
Builds upon: #284 
Is followed by: #244 

They should not just be treated as if they hovered with the keyboard,
but they have their own dedicated style and behavior.

### CODE REVIEW

File: tokens/properties/components/OptionsMenu.json [hunk 1]
```diff
@@ -49,6 +49,27 @@
 					"value" : "{background.color.base.active.value}"
 				}
 			},
+            "selected":{
```
Comment (Ladsgroup) [inline tokens/properties/components/OptionsMenu.json:52]: The spacing is wrong :D

Comment (SaiSan-WMDE) [reply tokens/properties/components/OptionsMenu.json]: Spacing issues were fixed in a recent [commit](https://github.com/wmde/wikit/pull/288/commits/8c2d68fb26388c80fc2a12312ad2f955ab07ea93)

**Review by itamargiv — APPROVED** [review]
LGTM, thank you!

File: vue-components/src/components/OptionsMenu.vue [hunk 2]
```diff
@@ -180,6 +188,25 @@ $base: '.wikit-OptionsMenu';
...
 
+		&--selected {
+			#{$base}__item__label, #{$base}__item__description {
+				color: $wikit-OptionsMenu-item-selected-color;
+			}
+		}
+
+		&--selected,
+		&--selected:hover,
+		&--selected#{$base}__item--hovered, {
```
Comment (guergana) [inline vue-components/src/components/OptionsMenu.vue:199]: I am getting some linting notification in VSCode about this comma at the end

File: vue-components/src/components/OptionsMenu.vue [hunk 3]
```diff
@@ -180,6 +188,25 @@ $base: '.wikit-OptionsMenu';
...
+			background-color: $wikit-OptionsMenu-item-selected-background-color;
+		}
+
+		&--selected:hover,
+		&--selected#{$base}__item--hovered, {
```
Comment (guergana) [inline vue-components/src/components/OptionsMenu.vue:204]: same here

**Pull request merged**